Common/include/config_structure.hpp

```diff
@@ -4972,197 +5153,961 @@ class CConfig {
-	 * \return The outlet pressure.
-	 */
-	void SetInflow_Pressure(unsigned short val_imarker, su2double val_fanface_pressure);
+  /*!
+   * \brief Get the back pressure (static) at an outlet boundary.
+   * \param[in] val_index - Index corresponding to the outlet boundary.
+   * \return The outlet pressure.
+   */
+  su2double GetInflow_MassFlow(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  su2double GetBleed_Temperature_Target(string val_marker);
+  su2double GetInflow_ReverseMassFlow(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  su2double GetBleed_Temperature(string val_marker);
+  su2double GetInflow_TotalPressure(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  void SetBleed_Temperature(unsigned short val_imarker, su2double val_bleed_temp);
+  su2double GetInflow_Temperature(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  void SetExhaust_Temperature(unsigned short val_imarker, su2double val_exhaust_temp);
+  su2double GetInflow_TotalTemperature(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  su2double GetExhaust_Temperature(string val_marker);
+  su2double GetInflow_RamDrag(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  su2double GetBleed_MassFlow_Target(string val_marker);
+  su2double GetInflow_Force(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  su2double GetBleed_MassFlow(string val_marker);
+  su2double GetInflow_Power(string val_marker);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
    * \param[in] val_index - Index corresponding to the outlet boundary.
    * \return The outlet pressure.
    */
-  void SetBleed_MassFlow(unsigned short val_imarker, su2double val_bleed_massflow);
+  void SetInflow_Pressure(unsigned short val_imarker, su2double val_fanface_pressure);
   
   /*!
    * \brief Get the back pressure (static) at an outlet boundary.
```

Comment:
Thanks Tom, You are completely right. I'm working on the copy/paste comments, I have already some of them (the most critical for the engine bc) and step by step I'm planning to update all of them. For the time being, I don't have a good example for traditional engine boundary condition, the ideal scenery would be to find an open nacelle geometry (any idea?) to offer a meaningful nacelle with meaningful boundary values. I'll check if the NASA rotor 37 has a nacelle.